sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    lambda_client = boto3.client("lambda")
</pre><p>As I pointed before, we can use lambda_helper's <code>lambda</code>  object to delete lambda function, so that we won't need to create a separate boto3 client to delete it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Fixed